zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZeppelinSparkClusterTest24.java

```diff
@@ -33,7 +33,7 @@ public ZeppelinSparkClusterTest24(String sparkVersion, String hadoopVersion) thr
   @Parameterized.Parameters
   public static List<Object[]> data() {
     return Arrays.asList(new Object[][]{
-            {"2.4.7", "2.7"}
```

Comment:
according to the name of the file this test is for 2.4 version
There is return Arrays.asList(new Object[][]{
{"3.0.3", "2.7"},
in SparkIntegrationTest30.java